pkg/kube/longhorn-generate-support-bundle.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+LOG_DIR=/persist/newlog/kube
+
+echo "Apply longhorn support bundle yaml at $(date)"
+
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: longhorn.io/v1beta2
+kind: SupportBundle
+metadata:
+  name: support-bundle-collect-info
+  namespace: longhorn-system
+spec:
+  description: collect-info
+  issueURL: ""
+  nodeID: ""
+EOF
+
+ready=0
+while [ $ready -ne 1 ]; do
+    state=$(kubectl -n longhorn-system get supportbundle.longhorn.io/support-bundle-collect-info -o json | jq -r .status.state)
+    if [ "$state" = "ReadyForDownload" ]; then
+        ready=1
+        break
+    fi
+    sleep 10
+    progress=$(kubectl -n longhorn-system get supportbundle.longhorn.io/support-bundle-collect-info -o json | jq -r .status.progress)
+    echo "support bundle progress percentage: $progress"
+done
+
+ip=$(kubectl -n longhorn-system get supportbundle.longhorn.io/support-bundle-collect-info -o json | jq -r .status.managerIP)
+filename=$(kubectl -n longhorn-system get supportbundle.longhorn.io/support-bundle-collect-info -o json | jq -r .status.filename)
+
+old_log_count=$(find "$LOG_DIR" -type f -name "longhornsupportbundle_*" | wc -l)
```

Comment:
I don't know how much log files the log directory can contain, but the find command it's being running twice, first to just count the number of files and again to remove all files except the 4 newer or older? The same logic can be achieved with one single run:

```sh
find "$LOG_DIR" -type f -name "longhornsupportbundle_*" | sort -n | head -n -4 | xargs rm -f
```

If there is no complex directory tree structure, `ls -c` can also be used to list files ordered by newer -> older...

Reply:
we're deleting all but the newest 4, I can condense this block down